sk0-back/src/users/services/password.service.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { PasswordService } from './password.service';
+
+describe('PasswordService', () => {
+  let service: PasswordService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [PasswordService],
+    }).compile();
+
+    service = module.get<PasswordService>(PasswordService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
+  });
+
+  describe('hash', () => {
+    it('should hash a password', async () => {
+      const password = 'examplePassword';
+      const hashedPassword = await service.hash(password);
+      expect(hashedPassword).toBeDefined();
+      expect(hashedPassword.length).toBeGreaterThan(0);
+    });
```

Comment:
This is a good test, if you want you can extend it with another `expect` to see that the original password does not equal to the `hashedPassword`. We should not use a hashing which has the same input and output.

```suggestion
      expect(hashedPassword.length).toBeGreaterThan(0);
      expect(hashedPassword).not.ToEqual(password);
    });
```

Reply:
good point, i'm on it